rca/utils/templatetags/util_tags.py

```diff
@@ -68,7 +68,28 @@ def social_media_links(context):
         "type": "linkedin",
         "label": "Linkedin",
     }
-    return [twitter, facebook, instagram, youtube, linkedin]
+    tiktok_url = (
+        "https://tiktok.com/@%s" % settings["utils"]["SocialMediaSettings"].tiktok
+        if settings["utils"]["SocialMediaSettings"].tiktok
+        else False
+    )
+    tiktok = {"url": tiktok_url, "type": "tiktok", "label": "TikTok"}
+    pinterest_url = (
+        "https://pinterest.com/%s" % settings["utils"]["SocialMediaSettings"].youtube
```

Comment:
Q: Is this set to `.youtube` as the accounts are have the same name or should it be changed to a pinterest specific field? I'm asking more out of curiosity than anything else : )

Reply:
Good spot! Thanks! It was a copy-paste, forgot to edit after pasting :see_no_evil: 

Fixed in 81612b38b35760b9d21f5eb475100229927b7dc3